ui/store/actions.ts

```diff
@@ -3696,6 +3696,7 @@ export function approvePermissionsRequest(
       if (err) {
         dispatch(displayWarning(err));
       }
+      forceUpdateMetamaskState(dispatch);
```

Comment:
why is this now added in?

Reply:
Once the first permission request is accepted we create a second approval, if we do not update the MetaMask state we won't get that second approval added to the UI state until the next sync. With this, we decrease the latency as we get the state almost instantly and it feels a lot faster!